src/Templating/Directory.Build.targets

```diff
@@ -8,10 +8,10 @@
       By default, all projects which produce packages are not intended to ship to NuGet.org as a product package.
       Packages which are intended to ship to NuGet.org must opt-in by setting this to true in the project file.
     -->
-    <IsProductPackage Condition=" '$(IsProductPackage)' == '' ">false</IsProductPackage>
+    <IsShippingPackage Condition=" '$(IsShippingPackage)' == '' ">false</IsShippingPackage>
```

Comment:
Suggest moving these properties to one file and importing that from here, the root `Directory.Build.targets` file and anywhere else they're needed

Reply:
What's the connection?

Reply:
src/Templating/ still builds as if it was a git submodule in its own repo. This file doesn't import the parent Directory.Build.targets file.

Reply:
I understand that. But, it _could_ import a new centralized file that's used elsewhere. Then again, I get that the `.targets` duplication is only between two files and that'll go away when Templating does import the root `Directory.Build.targets` file

Reply:
I tried doing that a while ago, but it made things more complicated. There are too many assumptions in both src/Templating/ and the repo-root D.B.targets which conflict. It was easier to keep these two projects in separate scopes with a little code duplication while we work on finishing up #4246.